packages/legends/index.d.ts

```diff
@@ -93,4 +94,6 @@ declare module '@nivo/legends' {
     }
 
     export type QuantileLegendSvg = React.FunctionComponent<QuantileLegendProps>
+
+    export const BoxLegendSvg: React.FC<LegendProps & Dimensions>
```

Comment:
Looking into `https://github.com/plouc/nivo/blob/master/packages/legends/src/svg/BoxLegendSvg.js` Proptypes I don't think these match.

Reply:
Which ones specifically?

Reply:
https://github.com/plouc/nivo/blob/master/packages/legends/src/svg/BoxLegendSvg.js#L113

You don't pass `height` & `width` but `containerWidth` & `containerHeight`

Reply:
Good catch, thank you.